src/v1/core.ts

```diff
@@ -18,8 +18,9 @@ import {
 } from './interfaces.compatible';
 import * as v20220901 from './interfaces.v20220901';
 import * as v20221101 from './interfaces.v20221101';
+import * as v20230901 from './interfaces.v20230901';
 
-type APIVersion = '2022-09-01' | '2022-11-01';
+type APIVersion = '2022-09-01' | '2022-11-01' | '2023-09-01';
```

Comment:
It might be beneficial to use a more scalable approach for handling different API versions, as the current approach might become unwieldy as more versions are added. Consider using a strategy pattern or a similar design pattern. [medium]